### PR TITLE
work section heading & corrected typo in Hackathon page

### LINKED ISSUE #571: [bug] Spelling error in Events page
### Describe the bug

There is a spelling error in Kunal's designation on the Events page.

### To Reproduce

1. Go to events page.
2.  scroll and select ongoing events
3.  scroll to judges section

### Expected Behavior

Typo in spelling of Manger. It should be Manager

### Screenshot/ Video

image

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sid-talesara**
## Fixes Issue

This PR fixes the following issues:

closes #571 

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### Section heading before changes:
image

### Changes Made: Changed the color of section headline
image

### Before changes - Typo => DevRel Manger @Civo
image

### After Changes 
image

### CODE REVIEW

**Comment by AnuragThePathak** [comment]
You have erased the changes for Heading color change in some commit.

**Comment by sid-talesara** [comment]
> You have erased...

Restored the heading color change.

**Review by AnuragThePathak — APPROVED** [review]

**Comment by AnuragThePathak** [comment]
@Chandraprakash-Darji is the heading color change fine?

**Review by Sandeep-RN — APPROVED** [review]
LGTM

**Comment by Chandraprakash-Darji** [comment]
@sid-talesara make color `text-content` becuse it should be  consistent all over the website

File: src/pages/PerticularhackathonPage.jsx [hunk 1]
```diff
@@ -236,7 +236,7 @@ const PerticularhackathonPage = ({ content }) => {
             <SpeakerCard
               {...{
                 name: 'Kunal Kushwaha',
-                description: 'DevRel Manger @Civo',
+                description: 'DevRel Manager @Civo',
```
Comment (Chandraprakash-Darji) [inline src/pages/PerticularhackathonPage.jsx:239]: undo this change, make this PR for the color fix only and also there is PR for this change already

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Review by Vivekgaddigi — APPROVED** [review]

**Comment by AnuragThePathak** [comment]
@Chandraprakash-Darji it seems the contributor is no longer active. Would you be able to update the PR? You may also create a new PR as per your convenience.

**Pull request closed without merging**